tests/test_cli.py

```diff
@@ -160,14 +160,6 @@ def test_main_fetch(fetch):
     args = cli._parse_args(argv)
     cli._execute(args)
 
-    # no land available for back extension
-    argv = ['monthly', '--startyear', '1980', '--endyear', '1980',
-            '--variables', 'total_precipitation', '--synoptic',
-            '--ensemble', '--land']
-    args = cli._parse_args(argv)
-    with pytest.raises(AssertionError):
-        cli._execute(args)
-
```

Comment:
While this test was a bit ill-described, I do think it is important to keep it, in a slightly modified form. We need to raise an error if the `--land` and `--prelimbe` flags are used together, as there is no back extension for era5-land.

Reply:
@Peter9192, thanks for the suggestion, I looked into it. So, if I understand correctly this particular test was more about `--land` being incompatible with the years prior to 1981, not back extension as comment says. The incompatibility of `--prelimbe` and `--land` are tested in `test_fetch.py`, because it is the place that gives you the `ValueError`, if those two tags are used together. As highlighted in #130, the request with `--ensemble` and `--land` actually goes through but fails on the `cdsapi` end. My suggestion is, for this pull request delete this test, and I will open a new PR with adding the `AssertionError` in case `--ensemble` and `--land` are used together. Then, I will re-add this test, but testing the `--ensemble` and `--land`. What do you think? @bvreede what's your opinion?

Reply:
Right! So I did notice that there was a discrepancy between the description (land and back extension not compatible) and the actual test (land not available for years prior to 1981). But you're right, I overlooked that there's also the issue of `--land` and `--ensemble` being incompatible. Your suggested approach sounds good to me 👍

Reply:
btw @bvreede is on holiday until next week

Reply:
Thanks for figuring this out; indeed, that's a good idea, let's do it!